src/pages/tamogatas.hu.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import Link from 'gatsby-link';
+import Layout from '../components/layout';
+import TabNav from '../components/donate/TabNav';
+import SocialMediaLinks from '../components/SocialMediaLinks';
+import './css/donate.css';
+
+export default () => (
+  <Layout>
+    <div className="donate-grid-container">
+      <div className="comingsoon-logo-container"></div>
+      <div className="donate-grid-item">
+        <div className="donate-title">
+          <span>Támogasd a Democrable-t!</span>
+        </div>
+        <div className="donate-description">
+          <span id="countdown">Azért küzdünk, hogy mindenki átlátható formában hozzáférjen bármely híreket közlő médium információihoz, így erősítve a médiatudatosságot és az átláthatóságot, ezzel pedig a demokráciát.</span>
+        </div>
+        <div className="donate-link-container">
+          <span><i className="icon material-icons">expand_more</i><a id="expand-more" onClick={() => {
+            var i;
+            var tablinks = document.getElementsByClassName('tablinks');
+
+            for (i = 0; i < tablinks.length; i++) {
+              tablinks[i].classList.remove('active')
+            }
+            var topPos = document.getElementById('contribute').offsetTop;
+            document.documentElement.scrollTop = topPos - 160;
```

Comment:
Also, a better solution would be to pass the information about which link is "active" to the TabNav as a prop.
This would require a larger rework, and we are pressed by time so let's leave it for now and I'll raise a ticket.

Reply:
The bigger problem is that scrolling doesn't change which one is active.

Reply:
The original version (before Gasby+React) has a solution for the bigger problem. It's working currently, but isn't the most beautiful code I've ever seen.